### PR TITLE
ShadowRient Network

### PR DESCRIPTION
**Pull request by diphie**
### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [x] No changes were made to the file's formatting.
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id.
* [x] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. of address `my.server`)*
    - You do not need to specify sub-domains, Lunar Client services automatically detect them.

### Logo Additions or Updates:
* [x] My image is a `png` file.
* [x] I uploaded my image to the `logos` directory.
* [x] My image has a transparent background and is square (1:1 aspect ratio).
* [x] My image is `512` pixels in width and height.
* [x] My image's file name matches its mapping id.

### CODE REVIEW

**Comment by Diphioc** [comment]
new

File: servers.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
-[ 
+[   {
```
Comment (GrowlyX) [inline servers.json:1]: Please drop the second bracket to a new line.

File: servers.json [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+        "id": "shadowrient",
+        "name": "Shadowrient Network",
+        "addresses": [
+            "mc.shadowrient.ml"
```
Comment (GrowlyX) [inline servers.json:5]: Remove the `mc.` prefix from this domain.

File: servers.json (hunk 1 above)
Comment (Diphioc) [reply servers.json]: Changed!

File: servers.json (hunk 2 above)
Comment (Diphioc) [reply servers.json]: changed!

**Comment by Radi0o** [comment]
updated

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please resolve the current merge conflicts

**Comment by itsJordan** [comment]
Closing due to inactivity.

**Pull request closed without merging**